geode-redis/src/main/java/org/apache/geode/redis/internal/pubsub/PubSub.java

```diff
@@ -81,19 +81,19 @@ long publish(
   long punsubscribe(GlobPattern pattern, Client client);
 
   /**
-   * Return a list of channel names that a client has subscribed to
+   * Return a list of channel names or patterns that a client has subscribed to
    *
    * @param client the Client which is to be queried
-   * @return the list of channels
+   * @return the list of channels or patterns
    */
-  List<byte[]> findSubscribedChannels(Client client);
+  List<byte[]> findSubscriptionsNames(Client client, Subscription.Type type);
 
   /**
-   * Return a list of pattern names that a client has subscribed to
+   * Return a list of channel names and patterns that a client has subscribed to
    *
    * @param client the Client which is to be queried
-   * @return the list of patterns
+   * @return the list of channels and patterns
    */
-  List<byte[]> findSubscribedPatterns(Client client);
+  List<byte[]> findSubscriptionsNames(Client client);;
```

Comment:
Only if you need to push another change: I think this should be `findSubscriptionNames` and the double `;` at the end of the line. No biggy.

Reply:
Yes!  Thank you for pointing this out!